pallets/msa/src/mock.rs

```diff
@@ -98,6 +98,8 @@ impl sp_std::fmt::Debug for MaxSchemaGrantsPerDelegation {
 	}
 }
 
+pub const NUMBER_OF_BUCKETS: u32 = 2; // so this can be used in genesis
```

Comment:
Not sure we need any of the changes to the mock. Rebase issue?

Reply:
Yes, this one should go away